index.html

```diff
@@ -47,11 +49,172 @@ <h1 class="h-title">I’m ROBERTO. Glad to see you!</h1>
+            </li>
+            <li class="tag">
+                <a href="" class="tag-in">JavaScript</a>
+            </li>
+        </ul>
+  
+        <a href="#" class="button project">See Project</a>
+         </div>
+  
+        
+  
+        </div>
+        <div class="card works">
+  
+      <img src="img/Snapshoot1.png" alt="a_screentshot" class="pic">
+  
+      <div class="text-card">
+        <h2 class="titleh2"> Multi-Post Stories</h2>
+          <ul class="canopy">
+            <li class="cano"><a href="#" class="can">CANOPY</a></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><a href="#">2015</a></li>
+          </ul>
+  
+      <p class="text">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+  
+      <ul class="ul">
+          <li class="tag">
+              <a href="">HTML</a>
+          </li>
+          <li class="tag">
+              <a href="" >CSS</a>
+          </li>
+          <li class="tag">
+              <a href="">JavaScript</a>
+          </li>
+      </ul>
+  
+      <a href="#" class="button project">See Project</a>
+       </div>
+  
       
-     
-      <section>
+  
+      </div>
+      <div class="card works">
+  
+    <img src="img/Snapshoot2.png" alt="a_screentshot" class="pic">
+  
+    <div class="text-card">
+        <h2 class="titleh2">Tonic</h2>
+        <ul class="canopy">
+          <li class="cano"><a href="#" class="can">CANOPY</a></li>
+          <li><a href="#">Back End Dev</a></li>
+          <li><a href="#">2015</a></li>
+        </ul>
+  
+    <p class="text">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+  
+    <ul class="ul">
+        <li class="tag">
+            <a href="">HTML</a>
+        </li>
+        <li class="tag">
+            <a href="" >CSS</a>
+        </li>
+        <li class="tag">
+            <a href="">JavaScript</a>
+        </li>
+    </ul>
+  
+    <a href="#" class="button project">See Project</a>
+     </div>
+  
+    
+  
+    </div>
+    <div class="card works">
+  
+    <img src="img/Snapshoot3.png" alt="a_screentshot" class="pic">
+  
+    <div class="text-card">
+      <h2 class="titleh2"> Multi-Post Stories</h2>
+        <ul class="canopy">
+          <li class="cano"><a href="#" class="can">CANOPY</a></li>
+          <li><a href="#">Back End Dev</a></li>
+          <li><a href="#">2015</a></li>
+        </ul>
+  
+    <p class="text">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+  
+    <ul class="ul">
+        <li class="tag">
+            <a href="">HTML</a>
+        </li>
+        <li class="tag">
+            <a href="" >CSS</a>
+        </li>
+        <li class="tag">
+            <a href="">JavaScript</a>
+        </li>
+    </ul>
+  
+    <a href="#" class="button project">See Project</a>
+     </div>
+  
+    
+  
+    </div>
+    </section>
```

Comment:
- [ ] Great work implementing your Works section here as requested by the previous reviewer 💯. However, I noticed you built your `<section class="main_wrapper">` including your headline too(the section with the phrase I'm Roberto Glad to see you!). The headline is a different section that should be outside the Works section. 🤓 
Below is an example of how the structure should look 👇🏽

```
<html>
    <head>
        <title>Portfolio</title>
    </head>
    <body>
        <header>
            Here goes logo, navigation, etc.
        </header>
        <main>
            <section class="headline-section">
                Here you can include your headline
            </section>
            <section class="works-section">
                Here you can include the work cards
            </section>
           <section class="about-me-section">
                Here you can include about myself, etc.
            </section>
        </main>
        <footer>
            Footer information, links, etc.
        </footer>
    </body>
</html>
```